source/style/style.css

```diff
@@ -0,0 +1,213 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
```

Comment:
- [ ] Please don't disable the linters and try to fix the problem but not to disable it because the linters is important to make your code follow best practices 🚀💯🔥🙏🏻